ros/resources/hosts.py

```diff
@@ -14,18 +14,18 @@ def get(self):
                     'id': '12345-57575757',
                     'account': '12345',
                     'vm_uuid': '12345a1',
-                    'state': 'Oversized',
-                    'recommendation_count': 4,
+                    'state': 'Crashloop',
```

Comment:
Is `Oversized` no longer a state to be checked?

Reply:
State value should be based on scoring, right? I have modified the scoring for this record.